docs/deploy-hyperlane.mdx

```diff
@@ -1,113 +1,334 @@
+  ```
+    
+</details>
+
+
+### Deploy contracts
+
+Continuing in the same instance with your `HYP_KEY` sourced, run:
 
 ```bash
-hyperlane deploy kurtosis-agents
+hyperlane core deploy
 ```
 
-Follow the prompts and then click on the Kurtosis Cloud link output by the command.
-
-If this is your first time using Kurtosis Cloud, you'll need to sign in with Google and Kurtosis will provision a remote cloud instance for your validator and relayer in roughly 2 minutes.
+Use the arrows and enter to select your custom chain from the bottom of the mainnet list. 
+It will take about five minutes for all contracts to deploy.
+<details> 
+  <summary>
+    Under `$HOME/.hyperlane/chains` you will find a new folder named with your custom chain’s name, and a file named `addresses.yaml` within that folder
+  </summary>
+  ```yaml
+    staticMerkleRootMultisigIsmFactory: "0x6906cb4741d3E2322E9f9aA645DfC8AB6F122c47"
+    staticMessageIdMultisigIsmFactory: "0x3CE97a32d9C8294691cBd2baC09B078EDa75c429"
+    staticAggregationIsmFactory: "0x81f969fDBF48278Ce09685Ce48e03388B6785aF4"
+    staticAggregationHookFactory: "0x3d864A3c25F61E3c3A7d02e980453A6E1f0a92A6"
+    domainRoutingIsmFactory: "0xC4c01f7B03f0fFa77A0265C600dEF7Ad28BCa5A2"
+    proxyAdmin: "0xABb7175d5F123172E7B7Fa467CC9fE4C2FEdb942"
+    mailbox: "0x5F58d75A9caDE4e2b191313223978dF049f93b81"
+    interchainAccountRouter: "0x43c0745b0dE9Cb780816a24ddE63d79Ca99B5dE8"
+    interchainAccountIsm: "0x9C96dC8f4257413225d6B5C47d1afbafc39B269F"
+    validatorAnnounce: "0xE3bd39BF92DB385dE6313D6070b035bD934378CB"
+    testRecipient: "0xa58462b1943Be1469Ed58db690C78583BA34Fd2E"
+    ```
+</details>
+    
+### Send test message
+
+In the same instance, run:
 
-![Get excited](/img/kurtosis-get-excited.png)
+```bash
+hyperlane send message --relay
+```
 
-A prefilled configuration screen will pop up with all your relayer and validator information from Steps 1 and 2.
+Currently core self-relay only works from **an existing origin chain** (holesky is recommended) **to your custom chain**. When dispatch is successful, you should see that the message was self-relayed!
 
 :::note
-
-If you are having issues deploying agents on Kurtosis Cloud through the CLI, you can manually enter this information by:
-
-1. Signing into [Kurtosis Cloud](https://cloud.kurtosis.com/).
-1. Clicking `Run` on the [Hyperlane Package](https://cloud.kurtosis.com/gateway/ips/18-206-3-164/ports/9711/catalog/github.com%2Fkurtosis-tech%2Fhyperlane-package).
-1. Filling in the configuration modal.
-
+**🎉 Congrats! You have successfully sent a message to your custom chain**
 :::
 
-The last step will be to input your `Validator Key` from Step 2. The screenshots below show an example prefilled configuration.
+## 4) Warp Route: configure & deploy your custom chain’s warp route contracts
+
+Now that you have a Hyperlane mailbox and core contracts on your chain, it’s time to set up ERC-20 token bridging between your chain and any other Hyperlane chain.
```

Comment:
I disagree that this deploy guide should also cover a warp deployment. But if we did want that, we should de-dupe this with the `deploy-warp-route` page and drop use a common component to drop in the shared instructions